signbank/dictionary/adminviews.py

```diff
@@ -1226,229 +991,22 @@ def get_queryset(self):
 
             qs = qs.filter(query)
 
+        qs = queryset_glosssense_from_get('GlossSense', GlossSearchForm, self.search_form, get, qs)
+        # this is a temporary query_parameters variable
+        query_parameters = dict()
+        # it is saved to self.query_parameters after the parameters are processed
+        query_parameters = query_parameters_from_get(self.search_form, get, query_parameters)
```

Comment:
This could be condensed to `query_parameters = query_parameters_from_get(self.search_form, get, dict())` or even replace `dict()` with `{}`.

Reply:
No, the query parameters are passed along as a state....
There are more modifications later.
The {} format does not work correctly with dictionary_variable.keys()

Reply:
Also see my comment above.

All you do here is assign an empty dictionary to a variable and immediately after, you pass that variable to a function. At that point it is still only an empty dictionary. Finally you assign the return variable of function to that same variable, overwriting the initial value, which was an empty dictionary.

Reply:
Yes, I understand. But I am more comfortable doing it explicitly. Then it matters less if some code ends up being inserted before the function call but after creation of the query_parameters dict.

This is just passing the state around as argument and result. (This is a style thing.)